burr/cli/__main__.py

```diff
@@ -322,13 +333,19 @@ def import_from_file(file_path: str) -> ModuleType:
     required=False,
     help="Python file or fully qualified python module to import for custom serialization/deserialization.",
 )
+@click.option(
```

Comment:
Why do we need to change this? A little unclear what this does -- worth adding a bit more docs

Reply:
it provides the name of the action to test for the test case.

enables you to use output from somewhere else as input for another action easily.